templates/security/index.html

```diff
@@ -435,28 +430,40 @@ <h2>Helping you manage security</h2>
   </div>
   <div class="row u-equal-height">
     <div class="col-6 p-card">
-      <h3>Critical security fixes with zero downtime</h3>
-      <p>The Canonical Livepatch Service lets you apply kernel fixes in seconds, without restarting your Ubuntu 18.04 LTS, 16.04 LTS or 14.04 ESM systems. Fewer reboots removes the operational burden associated planned downtimes for upgrades. It also means improved service availability and systems that are more compliant and secure.</p>
-      <p><a href="/livepatch">Learn more about Canonical's Livepatch Service&nbsp;&rsaquo;</a></p>
+      <h3>Reduce downtime and unplanned work</h3>
+      <p>The Canonical Livepatch service eliminates the need for unplanned maintenance windows for high and critical severity kernel vulnerabilities by patching the Linux kernel while the system runs. Reduce fire drills while keeping uninterrupted service with Canonical Livepatch service for up to ten years.</p>
+      <p>
+        <a href="/security/livepatch">
+          Learn more about Canonical's Livepatch Service&nbsp;&rsaquo;
+        </a>
+      </p>
     </div>
     <div class="col-6 p-card">
       <h3>Be compliant and FIPS certified</h3>
       <p>Canonical has received FIPS 140-2, Level 1 certification for cryptographic modules in Ubuntu 18.04 LTS and Ubuntu 16.04 LTS and Common Criteria EAL2 certification for Ubuntu 16.04 LTS. These certifications, alongside further security measures and achievements, enable organisations to meet compliance requirements within regulated industries.</p>
```

Comment:
```suggestion
      <p>Developing and running workloads for high security and government regulated environments requires a long and expensive validation process. Reduce your accreditation timeline and pass on your validation costs with the FIPS 140 and Common Criteria certifications available with Ubuntu Advantage and Pro.</p>
```
Does not match copydoc

Reply:
good catch, that change wasn't highlighted